lib/Backend/arm/LowerMD.cpp

```diff
@@ -1554,6 +1557,8 @@ LowererMD::LowerEntryInstr(IR::EntryInstr * entryInstr)
             insertInstr);
     }
 
+    bool trashScratchRegister = false;
```

Comment:
Nit: rename to isScratchRegisterThrashed.
The bool being true right now implies it needs to be trashed...